src/wrap/mod.rs

```diff
@@ -105,9 +131,22 @@ mod tests {
             stream,
         ));
 
+        let wrapped = textwrap::Wrapper::new(WrapConfig::default().max_line_length)
+            .initial_indent(" ")
+            .subsequent_indent(" ").fill(TEST_STR);
+        // the string resulting from fill() has one whitespace in the empty line which
+        let wrapped = wrapped.replace("\n \n", "\n\n");
```

Comment:
:+1: very nice quirk

Reply:
The impl has no trailing whitespace, because emtpy newlines don't get wrapped. But since the test just throws the whole string into the `textwrap`, the empty line also receives that `subsequent_ident()`. Could be called a bug in `textwrap` in my opinion, but I think that just regards our special case of code here.

Reply:
Got mixed up, mea culpa :>